### PR TITLE
Debug App Store workflows

### PR DESCRIPTION
**Pull request by stevecassidy**
# Debug App Store workflows

## JIRA Ticket

None

## Description

Trying to get App Store build workflow working again.

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.
- [x] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

File: .github/workflows/appstore-testflight.yml [hunk 1]
```diff
@@ -29,9 +29,10 @@ jobs:
     needs: check_date
     if: ${{ needs.check_date.outputs.should_run != 'false' }}
     continue-on-error: false
-    env:
-      TURBO_TEAM: ${{ vars.TURBO_TEAM }}
-      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+    # Disable Turbo to see if that fixes the build - possible cache issue
+    # env:
+    #   TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+    #   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
```
Comment (PeterBaker0) [inline .github/workflows/appstore-testflight.yml:35]: Still necessary?

File: app/ios/App/App/Info.plist [hunk 2]
```diff
@@ -17,20 +17,20 @@
...
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>org.fedarch.faims3</string>
+				<string>au.edu.faims.fieldmark</string>
 			</array>
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>202409110011</string>
+	<string>202412012317</string>
```
Comment (PeterBaker0) [inline app/ios/App/App/Info.plist:33]: so these will need to be env variables or some other build replacement later?

**Review by PeterBaker0 — APPROVED** [review]
Haven't tested, but looks reasonable - just check out need to remove turbo cache - but your call.

File: app/ios/App/App/Info.plist (hunk 2 above)
Comment (stevecassidy) [reply app/ios/App/App/Info.plist:33]: they get changed by the prebuildConfig.sh script.

**Pull request merged**